cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractApiTest.java

```diff
@@ -935,4 +936,14 @@ public void testQualityAnalysis() throws Exception {
         ApiResponse result = cloudinary.api().resource(API_TEST, ObjectUtils.asMap("quality_analysis", true));
         assertNotNull(result.get("quality_analysis"));
     }
+
```

Comment:
@nitzanj , what are we expected to get when deleting a non-existing folder or a non-empty one?
Let's add a test